paper.tex

```diff
@@ -164,14 +164,15 @@ \section*{Introduction}
 to be less terse and (potentially) also allow for documentation in the
 form of comments. However, there is no standard format shared across
 simulators, and the details of the demographic models are not separated
-from other simulation parameters. The APIs use by individual simulators
+from other simulation parameters. The APIs used by individual simulators
 to define demographic models are also not appropriate as an interchange
 standard because of the dependency on both the programming language involved,
 and the specific details of the simulator in question.
 
 Here we present ``Demes'', a data model and file format
 specification for complex demographic models developed by the PopSim
-Consortium. The Demes data model precisely defines key parameters
+Consortium. \aprcomment{do we need ``developed by the PopSim Consortium''?}
```

Comment:
We need to get the fact that this is the "blessed" format by the PopSim Consortium in somewhere.

Reply:
Ok, fair. I'll leave it.